CIP-0010/registry.json

```
@@ -71,6 +71,10 @@
     "transaction_metadatum_label": 1694,
     "description": "Voltaire - additional metadata for governance, provided in a separate transaction"
   },
+  {
+    "transaction_metadatum_label": 1854,
+    "description": "Multi-signatures wallet registration"
```

```suggestion
    "description": "CIP-146 Multi-signature wallet registration"
```
incase of competing standards, adding the CIP numbers aids differentiation